src/dipdup/exceptions.py

```diff
@@ -46,19 +46,10 @@ def __str__(self) -> str:
         return self.__doc__
 
     def _help(self) -> str:
-        # FIXME: Indentation hell
-        prefix = '\n' + ' ' * 14
-        context = prefix.join(str(self.args))
-        if context:
-            context = '{prefix}{context}\n'.format(prefix=prefix, context=context)
-
         return """
             An unexpected error has occurred!
-              {context}
```

Comment:
Why dropping the context?

Reply:
This code was copypasted from `ReindexingRequiredError` by mistake. Firstly, the help message is not formatted (line 49). Secondly, it shouldn't be, as `DipDupError` has no `args` to format when not subclassed.